Lesson-3/src/Lesson3.java

```diff
@@ -0,0 +1,97 @@
+public class Lesson3 {
+
+    public static void printThreeWords (){
+        System.out.println("Orange\nBanana\nApple");
+    }
+
+    public static void checkSumSign (int a, int b){
+
+        if ((a+b) >= 0){
+            System.out.println("Сумма положительная");
+        } else {
+            System.out.println("Сумма отрицательная");
+        }
+    }
+
+    public static void printColor (int value){
+
+        if (value <=0){
+            System.out.println("Красный");
+        } else if (value > 0 & value <=100 ) {
```

Comment:
Это лишнее, предыдущая проверка уже исключает текущую. Посмотрим разбор, там это обсуждалось
```suggestion
        } else if (value <=100 ) {
```